dist/common/scripts/scylla_setup

```diff
@@ -167,6 +168,9 @@ def warn_offline(setup):
 def warn_offline_missing_pkg(setup, pkg):
     colorprint('{red}{setup} disabled by default, since {pkg} not available.{nocolor}', setup=setup, pkg=pkg)
 
+def is_rhel7_old_kernel():
```

Comment:
IIUC it has nothing specifc with Centos/RHEL but with an old kernel, if so, I'd use `is_old_kernel()` and avoid referring to specific platforms on both logic / logger-msg

Reply:
I think we had similar problem with very old Ubuntu like 14.04, but now it's EOL and we don't support them anymore, so only distro which has outdate kernel is RHEL7/CentOS7/OEL7 on our [officially supported distro list](https://opensource.docs.scylladb.com/stable/getting-started/os-support.html).
Maybe `is_redhat_variants_ver7_old_kernel()` describes more correctly but I thought it's too long name.